### PR TITLE
Feat: combine multiple lines into one block, add a new option --block_size

### PR DESCRIPTION
**Pull request by Ninzore**
Known issue:  some text is not replaced with translation
I'm really not familiar with BS, please have a look

### CODE REVIEW

**Comment by Ninzore** [comment]
Relative issue https://github.com/yihong0618/bilingual_book_maker/issues/360

**Comment by yihong0618** [comment]
thanks will take a look later~

**Review by yihong0618 — COMMENTED** [review]
Nice

**Comment by Ninzore** [comment]
I want to mention the resume function doesn't work in this PR, but the rest are seems to be working properly.

File: book_maker/loader/epub_loader.py [hunk 1]
```diff
@@ -126,21 +127,51 @@ def _make_new_book(self, book):
...
-        new_p = copy(p)
+    #     self.helper.insert_trans(
+    #         p, new_p.string, self.translation_style, self.single_translate
+    #     )
+    #     index += 1
 
+    #     if index % 20 == 0:
+    #         self._save_progress()
+
+    #     return index
```
Comment (yihong0618) [inline book_maker/loader/epub_loader.py:161]: this can be deteled?

Comment (Ninzore) [reply book_maker/loader/epub_loader.py:161]: Sure, I can do that

**Comment by yihong0618** [comment]
@Ninzore thanks

**Pull request merged**